client/scripts/controllers/server/threads.ts

```diff
@@ -79,8 +83,10 @@ class ThreadsController {
     });
   }
 
-  public async edit(proposal: OffchainThread, body?: string, attachments?: string[]) {
+  public async edit(proposal: OffchainThread, body?: string, attachments?: string[], readOnly?: boolean, privacy?: boolean) {
```

Comment:
can we wrap these args onto separate lines?

Reply:
👍